index.html

```diff
@@ -0,0 +1,95 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Divyansh Singh</title>
+   <link rel="stylesheet" href="index.css">
+</head>
+<body>
+    <div>
+    <header>
+    <nav class="header">
+       <ul class="list">
+           <li class="item" id="name"><a href="#">Divyansh Singh</a></li>
+        </ul>
+        <img id="header-image" src="resources/Union.png" alt="menu">
+    </nav>
+
+    </header>
+    <section>
+        <h1 class="heading" id="main">Divyansh Singh</h1>
+    <h2 class="heading" id="job">Software Developer</h2>
+        <p class="para">
+            I can help you build a product , feature or website.
+             Look through some of my work and experience!If you 
+             like what you see and have a project you need coded,
+            don’t hestiate to contact me.
+            
+        </p>
+       
+    </section>
+    <img src="resources/Devider.png" class="divider" alt="divider">
+    <section class="spec">
+       
+        <ul>
+            <li class="list-heading">Language</li>
+          
+            <li><a href="#">HTML</a></li>
+            
+            <li><a href="#">CSS</a></li>
+         
+            <li><a href="#">Javascript</a></li>
+      
+            <li><a href="#">Ruby</a></li>
+        </ul>
+       
+       
+        <ul>
```

Comment:
- [ ] Please try to match this section like the Figma and uses the font defined for the template

| Your design  | Figma template|
| ------------- | ------------- | 
| ![image](https://user-images.githubusercontent.com/53402600/170591387-49956753-2d6d-4fb3-b741-6efa231db321.png)       | ![image](https://user-images.githubusercontent.com/53402600/170591536-4e26e556-3ae0-4a85-93db-fa1b7a1cc115.png)         |